modules/persistent-storage-manila-install.adoc

```diff
@@ -0,0 +1,150 @@
+    verbs: ["get", "list", "watch", "create", "delete", "update"]
+  - apiGroups: [""]
+    resources: ["persistentvolumeclaims"]
+    verbs: ["get", "list", "watch", "update"]
+  - apiGroups: ["storage.k8s.io"]
+    resources: ["storageclasses"]
+    verbs: ["get", "list", "watch"]
+  - apiGroups: [""]
+    resources: ["events"]
+    verbs: ["list", "watch", "create", "update", "patch"]
+  - apiGroups: ["v1"]
+    resources: ["secrets"]
+    verbs: ["get", "list"]
+----
+
+. Bind the rules via ClusterRoleBinding:
++ 
+[source,yaml]
+----
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: manila-provisioner
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: manila-provisioner-role
+subjects:
+- kind: ServiceAccount
+  name: manila-provisioner-runner
+  namespace: default
+----
+
+. Create a new secret:
++ 
+[source,yaml]
+----
+apiVersion: v1
+kind: Secret
+metadata:
+  name: manila-secret <1>
+  namespace: default <2>
+data:
+  os-authURL: <base64 encoded OpenStack Keystone URL>
+  os-userName: <base64 encoded Manila username>
+  os-password: <base64 encoded password>
+  os-projectName: <base64 encoded OpenStack project (tenant) name>
+  os-domainName: <base64 encoded OpenStack Manila service domain>
+  os-region: <base64 encoded OpenStack region>
+----
+<1> The secret name will be referenced by the Manila volume's
+StorageClass.
+<2> The secret namespace will be referenced by the Manila 
+volume's StorageClass.
+
+. Create a new StorageClass:
++ 
+[source,yaml]
+----
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: "manila-share"
+provisioner: "externalstorage.k8s.io/manila"
+parameters:
+  type: "default" <1>
+  zones: "nova" <2>
+  protocol: "NFS" <3>
+  backend: "nfs" <4>
+  osSecretName: "manila-secret" <5>
+  osSecretNamespace: "default" <6>
+  nfs-share-client: "0.0.0.0" <7>
+----
+<1> The link:https://docs.openstack.org/manila/latest/admin/shared-file-systems-share-types.html[Manila share type]
+the provisioner will create for the volume. This field is optional,
+and defaults to `default`.
+<2> Set of Manila availability zones that the volume might be created
+in. This field is optional, and defaults to `nova`.
+<3> Protocol used when provisioning a share. Valid options are
+`NFS` and `CEPHFS`. This field is required. 
+<4> Backend share used for granting access and creating the
+`PersistentVolumeSource`. Valid options are `nfs` and `cephfs`.
+This field is required.
+<5> Name of the secret object containing OpenStack credentials.
+This field is required.
+<6> Namespace of the OpenStack credentials secret object. This field
+is optional, and defaults to `default`.
+<7> Default NFS client for the share exported. This field is optional,
+and is only used for the `NFS` protocol. Defaults to `0.0.0.0`. 
+
+. Start the provisioner itself. The following example uses a Deployment:
+
+[source, yaml]
+----
+kind: Deployment
+apiVersion: extensions/v1beta1
```

Comment:
As you mentioned this should be `apps/v1`.

Reply:
Corrected.